src/main/java/ua/okwine/productexpirationdate/dto/SupplierDTO.java

```diff
@@ -0,0 +1,18 @@
+package ua.okwine.productexpirationdate.dto;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Data
```

Comment:
Use @getter and @setter because @data generate also equals and hashcode, toString

@RequiredArgsConstructor works only with final fields. You have no final fields. Use AllArgsConstructor and NoArgsConstructor

Reply:
ok, fixed 
@DaTa -> @Getter @Setter ; 
@Requiredargsconstructor  -> @AllArgsConstructo @NoArgsConstructor